### PR TITLE
two-factor verification

### PR DESCRIPTION
**Pull request by vpetrusenko**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def draw(routes_name)
   get '/documents/websocket_api'
   get '/documents/oauth'
   resources :documents, only: [:show]
-  resources :two_factors, only: [:show, :index, :update]
+  #resources :two_factors, only: [:show, :index, :update]
```
Comment (dmk) [inline config/routes.rb]: just remove this line

File: config/routes.rb [hunk 2]
```diff
@@ -109,7 +109,7 @@ def draw(routes_name)
       end
     end
 
-    post '/pusher/auth', to: 'pusher#auth'
+   ; post '/pusher/auth', to: 'pusher#auth'
```
Comment (dmk) [inline config/routes.rb]: I think it's wrong :)

File: config/routes/admin.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@
       post :toggle
     end
 
-    resources :two_factors, only: [:destroy]
+    #resources :two_factors, only: [:destroy]
```
Comment (dmk) [inline config/routes/admin.rb]: remove

**Review by dmk — CHANGES_REQUESTED** [review]
Looks good, but needs to be updated
Also please remove two-factor tests

**Comment by yivo** [comment]
@vpetrusenko have you removed gem dependencies related to this feature?

**Comment by yivo** [comment]
@vpetrusenko Also could you please check that tests are not broken?

**Comment by yivo** [comment]
@vpetrusenko Why there are 2 the same PRs? https://github.com/rubykube/peatio/pull/25

**Comment by vpetrusenko** [comment]
@yivo sorry have some troubles with this PR , make some changes in new PR

**Pull request closed without merging**